dotnet/src/SemanticKernel.Core/Functions/SKFunctionFromPrompt.cs

```diff
@@ -24,7 +24,7 @@ namespace Microsoft.SemanticKernel;
 /// A Semantic Kernel "Semantic" prompt function.
 /// </summary>
 [DebuggerDisplay("{DebuggerDisplay,nq}")]
-internal sealed class SKFunctionFromPrompt : ISKFunction
+internal sealed class SKFunctionFromPrompt : KernelFunction
```

Comment:
Can we rename these derived types as well? e.g. KernelFunctionFromPrompt, PromptKernelFunction, etc.

Reply:
Yes, but let's do this as a seperate PR to keep this one to a reasonable size and number of changes